src/Components/Timeline.js

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from "react";
+import { getPostsList } from "../Services/api.services";
+import Post from "./Post";
+import { Content, Heading } from "../Styles/MainPage";
+import styled from "styled-components";
+
+export default function Timeline() {
+    const [statusMessage, setStatusMessage] = useState("Loading");
+    const [postsList, setPostsList] = useState([]);
+
+    function checkIfPostsListIsEmpty() {
+        if (postsList === []) setStatusMessage("Nenhum post encontrado");
+        else setStatusMessage("OK");
+    }
+
+    useEffect(() => {
+        getPostsList("2b19cbc1-856a-4d00-9666-2676d198c4bd")
```

Comment:
Uma coisa que seria uma boa para as próximas tarefas deixarem o mockup em um context colocar o context aplicado por toda a aplicação, assim quando for necessário substituir pelos dados reais enviados pelo login vai ficar mais tranquilo no futuro.